src/components/atom/AppNextButton.tsx

```diff
@@ -23,7 +23,10 @@ const AppNextButton: FC<ButtonProps> = ({title, disabled = false, bgColor = "bg-
     );
     return (
         <button
-            className={btnClasses} disabled={disabled} type="submit">
+            className={btnClasses} disabled={disabled} {...(onClick
```

Comment:
```
{...(onClick ? {type: "button", onClick} : {type: "submit"})}
```

What do you think if we extract this in a variable to have the component clean?

My suggestion in this case is:
```
let buttonType = onClick ? {type: "button", onClick} : {type: "submit"};
    return (
        <button className={btnClasses} disabled={disabled} {...buttonType}>
            {title}
        </button>
    );
```